.github/workflows/integration-tests.yaml

```diff
@@ -1,11 +1,25 @@
-name: Performance benchmark
+name: Integration Tests
 
 on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [ master ]
   workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        maven_profiles: [
+            "jackson-classic", "jsonb-classic",
+            "jackson-reactive", "jsonb-reactive",
+            "jackson-classic,quarkus-1.4", "jsonb-classic,quarkus-1.4",
+            "jackson-reactive,quarkus-1.11", "jsonb-reactive,quarkus-1.11"
```

Comment:
Why `11` and not latest  (`13`)?

Reply:
1.11 is the oldest quarkus for which we support reactive version, I moved it from 1.12 few days ago when I run the tests in 1.11 and it worked :)

Reply:
Regarding 1.13: it is used when no `quarkus-XX` profile is specified